### ISSUE TITLE
Blank in selecting image

### ISSUE THREAD

**Issue by AlexVincent525** [Sh1d0w/multi_image_picker#147]
Hi, it's me again.
When I was testing on iOS devices, there's a weird bug appear when I'm selecting images, which is some of the image container was blank and displayed as a background color container, but little of them is normal.
As I scroll quickly, those images will appear at the moment and disappear in maybe 50-100ms, so it looks like those images were loaded correctly but not displayed normally.
Android seems not related with this.
Now I'm using flutter `v1.5.4-hotfix.2`, multi image picker with `4.3.0`, but `4.2.1` has the same issue.
Hope you can help me investigate this issue.

Screenshot:
![image](https://user-images.githubusercontent.com/15884415/57583543-dc20ce80-7503-11e9-902e-77944a344d00.png)

Sample code:
```
await MultiImagePicker.pickImages(
    enableCamera: true,
    cupertinoOptions: CupertinoOptions(
        backgroundColor: currentColorValue,
        takePhotoIcon: "chat",
    ),
    materialOptions: MaterialOptions(
        actionBarColor: currentColorValue,
        statusBarColor: currentColorValue,
        actionBarTitle: "Choose Images",
        allViewTitle: "All Images",
    ),
);
```

**Comment by Sh1d0w** [comment]
Is this happening on a real device or simulator? If it is a simulator they are usually dependant on your mac resources and are laggy. I suggest you test on a real device.

I was not able to reproduce this, but either way this is the native BSImagePicker view. So in case you can constantly reproduce this I suggest you to open a bug report in the BSImagePicker official repository with all info to reproduce this, and I am sure the maintainer will look into it.

**Comment by AlexVincent525** [comment]
It's on real devices, and some of my users report this, but just a little of them.

**Comment by AlexVincent525** [comment]
Sorry, but which BSImagePicker you used? https://github.com/mikaoj/BSImagePicker or https://github.com/siralam/BSImagePicker

**Comment by Sh1d0w** [comment]
First one, there is also link to in in the README :)

**Comment by AlexVincent525** [comment]
Thanks, it's related to iCloud images. https://github.com/mikaoj/BSImagePicker/issues/126

**Comment by Sh1d0w** [comment]
@AlexVincent525 Did you open new issue in the repo?

I see there are several other reports about it like https://github.com/mikaoj/BSImagePicker/issues/212 , but the author of the issue was not very responsive to provide more info. If you do I am sure the mantainer will look into this.

**Comment by AlexVincent525** [comment]
@Sh1d0w No, cause it seems to have a solution rather than a new issue, so I'm asking that person in my last reply's issue to see if he has fixed it.


> [mikaoj/BSImagePicker#126](https://github.com/mikaoj/BSImagePicker/issues/126)

At here. Something like `PHContentEditingInputRequestOptions` might handle this.

**Comment by Sh1d0w** [comment]
I've released version 4.5.5 which have the fix from https://github.com/Sh1d0w/multi_image_picker/issues/163

I think that should solve the problem.
